safe_transaction_service/history/models.py

```diff
@@ -689,6 +694,9 @@ def build_from_trace(
         tx_type = InternalTxType.parse(trace["type"])
         call_type = EthereumTxCallType.parse_call_type(trace["action"].get("callType"))
         trace_address_str = self._trace_address_to_str(trace["traceAddress"])
+        detail_hash = fast_keccak_hex(
+            f'transfer_{trace["transactionHash"]}_{trace["traceAddress"]}'.encode()
```

Comment:
@Uxio0 `traceAddress` is going to be None in L2 chains? In that case, the issue propose to use `log_index`, but I can't find it in the model InternalTx.

Reply:
No, it's always there (for L2 is generated making sure it's always unique and deterministic)